kateryna/databases.py

```diff
@@ -0,0 +1,113 @@
+class PostgreSQLConnectionRefusedError(Exception):
+    def __init__(self, message):
+        self.message = message
+
+
+class Connection(object):
+    def __init__(self, db_name):
+        self.closed = False
+        self.db_name = db_name
+
+    def __str__(self):
+        return "Connection to {}".format(self.db_name)
+
+
+class BaseFramework(object):
+    allowed_commands = ["SELECT", "INSERT", "UPDATE", "DELETE"]
+
+    def __init__(self, connection, **kwargs):
+        self.connection = connection
+        for attr, value in kwargs.items():
+            setattr(self, attr, value)
+
+    def connect(self):
+        self.connection.closed = False
+        return self.connection
+
+    def execute(self, command):
+        if not command.upper() in self.allowed_commands:
+            raise Exception("Provide valid command")
```

Comment:
It makes sense to add some custom error handling too.

Something similar to:

```
class BaseExceptionHandler(Exception):
    pass
```
And re-use it here.

Also, please use full sentences everywhere. ( dot is missing in the end of sentence.)